src/cap-description.adoc

```diff
@@ -448,8 +448,8 @@ T[MW - 1:MW - 2] = B[MW - 1:MW - 2] + LCout + LMSB
 Decoding the bounds:
 
 ```
-top:    t = { a[MXLEN - 1:E + MW] + ct, T[MW - 1:0]    , {E{1'b0}} }
-base:   b = { a[MXLEN - 1:E + MW] + cb, B[MW - 1:0]    , {E{1'b0}} }
+top:    t = { 1'b0, a[MXLEN - 1:E + MW] + ct, T[MW - 1:0], {E{1'b0}} }
```

Comment:
Hi Tariq! Does this mean that the `top` is now 66 bits and `top[65]==1'b0` always? Not sure about this.

This pseudo-code was always meant to be temporary and it has no formal definition, but I seem to recall that `ct` could overflow `a[MXLEN - 1:E + MW]` and that's how we would get the 65 bits to form the top bound of the infinite cap bounds. This is why I suggested [here](https://github.com/riscv/riscv-cheri/issues/362#issuecomment-2622884049) that this be replaced with SAIL snippets given that those now work although this might make the cap encoding description a bit more difficult to read.

Reply:
yes - the top bound is wider, but we're only talking about the rep range check here which isn't affected by any bits above MXLEN-1